nessie/src/test/java/org/apache/iceberg/nessie/TestNessieCatalog.java

```diff
@@ -115,17 +117,16 @@ private void resetData() throws NessieConflictException, NessieNotFoundException
   private NessieCatalog initNessieCatalog(String ref) {
     NessieCatalog newCatalog = new NessieCatalog();
     newCatalog.setConf(hadoopConfig);
-    newCatalog.initialize(
-        "nessie",
-        ImmutableMap.of(
-            "ref",
-            ref,
-            CatalogProperties.URI,
-            uri,
-            "auth-type",
-            "NONE",
-            CatalogProperties.WAREHOUSE_LOCATION,
-            temp.toUri().toString()));
+    ImmutableMap.Builder<String, String> options =
+        ImmutableMap.<String, String>builder()
+            .put("ref", ref)
+            .put(CatalogProperties.URI, uri)
+            .put("auth-type", "NONE")
+            .put(CatalogProperties.WAREHOUSE_LOCATION, temp.toUri().toString());
+    if (apiVersion == NessieApiVersion.V2) {
```

Comment:
seems inconsistent with what's being done in other files where `client-api-version` is always added. Any particular reason to not just do `options.put("client-api-version", apiVersion == NessieApiVersion.V2 ? "2" : "1")`?

Reply:
updated it. 

Else if the default value is changed in the future, I have to update the test case, If I won't add it for both versions.